### PR TITLE
builddep: print errors from RPM SPEC parser

### PR DESCRIPTION
**Pull request by pspacek**

### CODE REVIEW

**Comment by mluscon** [comment]
@dnf-bot retest this please

File: plugins/builddep.py [hunk 1]
```diff
@@ -174,8 +174,9 @@ def _src_deps(self, src_fn):
     def _spec_deps(self, spec_fn):
         try:
             spec = rpm.spec(spec_fn)
-        except ValueError:
-            msg = _("Failed to open: '%s', not a valid spec file.") % spec_fn
+        except ValueError as ex:
+            msg = _("Failed to open: '%s', not a valid spec file: %s") % (
+                    spec_fn, ex)
```
Comment (j-mracek) [inline plugins/builddep.py]: What about to add:
```from dnf.i18n import ucd```
and
```spec_fn, ucd(ex)```
Just for case that something strange is in the spec file.

**Comment by j-mracek** [comment]
@dnf-bot r+

**Comment by pspacek** [comment]
Hmm, I cannot see any issue mentioned in http://ci.dnf.baseurl.org/job/dnf-plugins-core-update-gh-status/237/console

**Comment by ignatenkobrain** [comment]
@dnf-bot retry

**Comment by j-mracek** [comment]
@dnf-bot retry

**Pull request closed without merging**